doc/source/getting_started/index.rst

```diff
@@ -21,41 +23,42 @@ Install the latest release from `PyPi
 
    pip install ansys-fluent-core
 
-Alternatively, install the latest from `PyFluent GitHub
+Alternatively, install the latest release from `GitHub
 <https://github.com/pyansys/pyfluent/issues>`_ with:
 
 .. code::
 
    pip install git+https://github.com/pyansys/pyfluent.git
 
 
-For a local "development" version, install with:
+If you plan on doing local *development* of PyFluent with Git, install
+the latest release with:
 
-.. code::
+.. code:: console
 
    git clone https://github.com/pyansys/pyfluent.git
    cd pyfluent
+   pip install pip -U
    pip install -e .
+   python codegen/allapigen.py  # Generates the API files
 
-See the `README. <https://github.com/pyansys/pyfluent/blob/main/README.rst>`_
-for instructions on installing the PyFluent ``ansys-fluent-core`` package
-and modifying it locally. The changes that you make are reflected in your setup
-after you restart the Python kernel.
 
-Launching Fluent
-----------------
+Any changes that you make locally are reflected in your setup after you restart
+the Python kernel.
 
-You can launch Fluent from Python using the ``launch_fluent`` function:
+Launch Fluent
+-------------
+To launch Fluent from PyFluent, use the ``launch_fluent`` method:
 
 .. code:: python
 
   import ansys.fluent.core as pyfluent
   solver = pyfluent.launch_fluent(precision="double", processor_count=2, mode="solver")
   solver.check_health()
 
-Fluent is now active. You can send commands to it as a genuine Python class.
-For example, if you wanted to read a case file, update a setting, and iterate the
-solver:
+Once Fluent is active, you can use the ``session.solver.tui`` interface to send
+Fluent TUI commands to Fluent as a genuine Python class. For example, you can read a
```

Comment:
This is better but I believe that the retained "as a genuine Python class" is still clumsy and confusing (it's separated from the thing it's describing). We could just refer to the ``session.solver.tui`` Python interface and remove "as a genuine Python class". It's not surprising that a Python library has genuine Python classes.